app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Sign In</h1>
+<form method="POST" action="/sessions/login" enctype="multipart/form-data">
+<input type="hidden" name="authenticity_token" value="<%= Rack::Protection::AuthenticityToken.token(env['rack.session']) %>" />
+    <label for="email">Email</label>
+    <% if defined?(email) %>
```

Comment:
Passing in the user object rather than the email would make it possible to not have this logic here.